### ISSUE TITLE
Nav Base: Links do not announce selected state

### ISSUE THREAD

**Issue by nuh1234** [OfficeDev/office-ui-fabric-react#11811]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:
aria-selected is not being populated!

**Comment by khmakoto** [comment]
Hi @nuh1234, what is the scenario in which you would want `aria-selected` populated? Since links prompt navigation I feel like `aria-selected` would not add useful information given how on selection there would be some kind of navigation.

**Comment by smhigley** [comment]
this looks like a dupe of #11788 (`aria-selected` isn't supported on links, so the real issue is a lack of `aria-current`), and would be resolved by #11789

**Comment by khmakoto** [comment]
Thanks for noticing and letting me know @smhigley, that does seem to be the case! @nuh1234 could you confirm if #11788 is describing the same thing you're experiencing?

**Comment by HalcyonPride** [comment]
@khmakoto, speaking on behalf of @nuh1234: we concur with what @smhigley said 😊

**Comment by khmakoto** [comment]
Thanks @HalcyonPride, I'll be adding the `Resolution: Duplicate` tag in favor of #11788 then!
